testsuite/tests/ui/devel/auth/test_login_rhsso.py

```diff
@@ -61,6 +60,4 @@ def test_devel_login_rhsso(custom_devel_rhsso_login, navigator, testconfig):
     signup_view = SignUpView(navigator.browser)
```

Comment:
Next 2 lines should be removed as well since you are not working with SignupView in this test at all.